resources/js/contest-voting/entry.coffee

```diff
@@ -77,6 +70,23 @@ export class Entry extends React.Component
             target: '_blank'
             i className: 'fas fa-fw fa-lg fa-external-link-alt'
 
+
+  renderScore: ->
+    score = if @props.entry.results.score_std != null
+      @props.entry.results.score_std.toFixed(2)
```

Comment:
transChoice requires the number to be a number, not a string. It'll also formatNumber it for display already.

(or alternatively the string must be passed as `count_delimited` replacement... which I noticed there's no way to manually override it for the js version)

Reply:
occurred to me that `1.0` is `one point zero points`, not the singular, but I don't know how that rule applies in other languages...